test/Riok.Mapperly.Tests/Generator/IncrementalGeneratorTest.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Riok.Mapperly.Tests.Generator;
+
+[UsesVerify]
+public class IncrementalGeneratorTest
+{
+    private const string AddMappersStep = "ImplementationSourceOutput";
+    private const string ReportDiagnosticsStep = "diagnostics";
```

Comment:
I think it could be helpful to define these names as consts inside `Riok.Mapperly` and reference them from the MapperGenerator and here from the tests.